### DISCUSSION TITLE
Different coloured stacked columns

### DISCUSSION THREAD

**Discussion by ivanleong** [patrykandpatrick/vico#783] [Questions]
### Question

Is it possible to have stacked columns but with different colours in the same series depending on their values like so:

![Screenshot 2024-07-02 at 15 53 48](https://github.com/patrykandpatrick/vico/assets/1160053/464b1511-f4ab-453b-a8d5-a82a3a5e8979)

**Comment by Gowsky** [comment]
Hi @ivanleong, yes, it's possible, [see here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/column-layer).
I know that you're using Vico 2, based on your previous questions, but please fill the required information next time.
